### ISSUE TITLE
.table() create a table truncated

### ISSUE THREAD

**Issue by poloop** [cronvel/terminal-kit#189]
Hello,

I think the last version introduce an issue that truncate the table.

terminal-kit@2.1.6 work properly except there is lot of space after the table if fit is set to true.

**Comment by T3sT3ro** [comment]
repro for example with
```
[
  [ '92', ' 3', '88', '13', '50' ],
  [ '90', '70', '24', '28', '52' ],
  [ '15', '98', '10', '26', ' 5' ],
  [ '84', '34', '37', '73', '87' ],
  [ '25', '36', '74', '33', '63' ]
]
```

**Comment by cronvel** [comment]
@T3sT3ro Thanks for posting the example that breaks things up (no sample code = unlikely to be resolved).

It's fixed as of Terminal-Kit v2.2.4.

**Comment by cronvel** [comment]
@T3sT3ro @poloop Also you may want to pass the option `expandToWidth: false`, depending on if you want the table to use all the terminal's width or not.
